main/res/values/strings.xml

```diff
@@ -1586,8 +1586,8 @@
     <string name="init_geokrety_register_ok">Login successful</string>
     <string name="init_geokrety_register_fail">Login failed</string>
     <string name="init_geokrety_description">Authorize c:geo with GeoKrety.org to log GeoKrety trackables.</string>
-    <string name="init_geokrety_cache_description">GeokretyMap.org provides cached datas for GeoKrety trackables.\nAvailable datas contains more GeoKrety informations like \'description\' or \'missing\' status.\n\nDatas are not live!\nPositions: ~30min.\nDescriptions: ~24h.</string>
-    <string name="init_geokrety_cache">Use GeoKretyMap.org</string>
+    <string name="init_geokrety_cache">Use api.GeoKrety.org</string>
+    <string name="init_geokrety_cache_description">api.Geokrety.org provides cached data for GeoKrety trackables.\nAvailable data contains more GeoKrety informations like \'description\' or \'missing\' status.\n\nData is not live!\nPosition: ~30min.\nDescription: ~24h.</string>
```

Comment:
As GK and GKM are now one project, api.geokrety.org should be the default.

We may eventually keep the old fashion as a fallback (probably in case of GK api side service failure) or simply remove it.

Reply:
My question would go into the same direction:
Do we then still need this additional option to use a second platform to receive the data. I would imagine, we could completely remove the (optional) differentiation between GKM and GK in code and menu?!

Reply:
BTW, the timing set in the original message are now completely obsolete, there is just 1 minute max of delay. But we still have cached responses for 15 minutes (now I'm wondering if we should keep this cache or lower it a bit…)

Reply:
> Do we then still need this additional option to use a second platform to receive the data. I would imagine, we could completely remove the (optional) differentiation between GKM and GK in code and menu?!

api.geokrety.org has no direct access to the geokrety database, if I understand the change right.

My test case is
- open geokret (maybe open cache, open log dialog, open geokret from inventory)
- write and sent log
- refresh 
- the log shoud be in the log list

Currently without api.geokrety.org we have no log list, with api I have to refresh some times until the log will be visible. The log can be seen immediately on the geokrety website.
(With a TB I habe the log instantly after refresh)

I think we can remove the extra entry for api.geokrety(map).org from the options as first step in this PR.
In another PR the complete code conversion to api.gekrety org should be realized.

Reply:
Effectively, there is a little delay by design, but the cache of 15min may be a big problem. I'll disable it.

Reply:
> As GK and GKM are now one project, api.geokrety.org should be the default.

GKM was already default.
The isGeokretyCacheActive() should now always return true.

Reply:
GKM cache on select GK endpoint is now disabled. Ex https://api.geokretymap.org/gk/73512 should refresh the date attribute in gkxml element on each page refresh.